### DISCUSSION TITLE
Embed into a web application

### DISCUSSION THREAD

**Discussion by alexfdezsauco** [datalust/seq-tickets#1234] [Ideas]
Is there a way to "embed" seq into a web application as widget / iframe? 

Let's say I want to display the live logs stream or just the tail for quick view into my application. 

For more detailed analysis, Is always possible navigate to Seq, but could be useful this kind of integration.

**Comment by nblumhardt** [comment]
Hi @alexfdezsauco - thanks for getting in touch.

Seq's UI is not really set up for embedding, but it's usually quite straightforward to add a custom view of the latest events to your own web page using the Seq API, if that helps in your case?

There's an example snippet showing how to read events with the C# API client here:

https://github.com/datalust/seq-api#reading-events

Let me know if this helps, and if you need any more info.

Best regards,
Nick

**Reply by alexfdezsauco** [reply]
Yes, thanks. I'm already using it.

**Reply by nblumhardt** [reply]
That's great - thanks @alexfdezsauco 👍
